main.go

```diff
@@ -218,3 +253,85 @@ func addPodSecurityPrivilegedLabels(watchNamespaceName string) error {
 	}
 	return nil
 }
+
+func DoesCRDExist(CRDName string) (bool, error) {
```

Comment:
Instead, can we use the discovery client to look for the cloud credential-specific resource? 

https://pkg.go.dev/k8s.io/client-go/discovery

Reply:
Done, updated the PR.